fix(@angular/cli): fix tests to match app component template

cyrilletuzi opened the pull request:
Fixes #6537

clydin commented:
I think the actual problem is that the inline template is different.  It only includes the prefix and not the welcome text.

cyrilletuzi commented:
@clydin is right, I updated the PR.

By the way, as I'm modifying the template : 
- I added a comment about what can be delete in the template, it was confusing if routing,
- do we really want inline styles, even if it's a boilerplate ? especially when it's not really relevant visually, as on big screens it's really unaligned with the links below.

gioragutt commented on packages/@angular/cli/blueprints/ng/files/__path__/app/app.component.ts:
```diff
@@ -3,9 +3,38 @@ import { Component } from '@angular/core';
+      <h1>
+        Welcome to {{title}}!!
+      </h1>
+      <img width="300" src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4N
```
Isn't it better to save the file in the `assets` folder and reference it here? Would make the file a lot cleaner.

clydin replied:
The content is going to be almost immediately deleted so there's little point to adding an additional asset that must be deleted as well.

gioragutt replied:
@clydin notice below that I referenced your PR. I had to do a similar fix in my PR so that the tests will pass, so I've just fixed the `Welcome to {{title}}!` so that it would work and wouldn't have merge-conflicts with your PR if mine happens to go in master before yours.

cyrilletuzi commented:
Unfortunately I won't be available the next 2 weeks, so feel free to edit my PR or to create a new one.

cyrilletuzi commented:
Closing as I think it's outdated.
